### PR TITLE
Fix Anthropic Version Errors

### PR DESCRIPTION
**Pull request by phenomenal-manish**
*Issue #, if available:*

*Description of changes:*
1. Setting httpx==0.27.2 in setup.py
2. Remove count_tokens method for anthropic models.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     packages=find_packages(),
     install_requires=[
         "tqdm",
+        "httpx==0.27.2",
         "anthropic",
```
Comment (BardiyaAk) [inline setup.py]: can we remove our dependency to anthropic now?

Comment (phenomenal-manish) [reply setup.py]: Yes!

Comment (phenomenal-manish) [reply setup.py]: Added a new commit!

Comment (ndronen) [reply setup.py]: Was the new commit supposed to remove `anthropic` from `setup.py`? I don't see that change here.

Comment (phenomenal-manish) [reply setup.py]: Just added a new commit to address this!

**Review by BardiyaAk — CHANGES_REQUESTED** [review]
can you remove line 11 `import anthropic` too?

**Review by BardiyaAk — APPROVED** [review]
LGTM

**Review by ndronen — APPROVED** [review]
LGTM

**Pull request merged**